rule-engine/rule-engine-components/src/main/java/org/thingsboard/rule/engine/action/TbCopyAttributesToEntityViewNode.java

```diff
@@ -71,14 +75,14 @@ public void init(TbContext ctx, TbNodeConfiguration configuration) throws TbNode
 
     @Override
     public void onMsg(TbContext ctx, TbMsg msg) {
-        if (DataConstants.ATTRIBUTES_UPDATED.equals(msg.getType()) ||
-                DataConstants.ATTRIBUTES_DELETED.equals(msg.getType()) ||
-                DataConstants.ACTIVITY_EVENT.equals(msg.getType()) ||
-                DataConstants.INACTIVITY_EVENT.equals(msg.getType()) ||
-                SessionMsgType.POST_ATTRIBUTES_REQUEST.name().equals(msg.getType())) {
+        if (ATTRIBUTES_UPDATED.name().equals(msg.getType()) ||
```

Comment:
Let's create some utility methods in the TbMsg. For example, "msg.checkType(TbMsgType...)"